.github/workflows/codeql-query.yml

```
@@ -66,13 +66,17 @@ jobs:
         shell: node {0}
         run: |
           const fs = require('fs');
+          const process = require('process');
           const allRepos = JSON.parse(fs.readFileSync("instructions.json")).repositories;
           const repoNwos = new Set(${{ toJSON(matrix.repoNwos) }});
           const repositories = allRepos.filter(r => repoNwos.has(r.nwo));
           if ("${{ secrets.TEST_PAT }}") {
             allRepos.forEach(r => r.pat = "${{ secrets.TEST_PAT }}")
           }
-          console.log(`::set-output name=repositories::${JSON.stringify(repositories)}`);
+          const githubOutput = process.env.GITHUB_OUTPUT;
+          const outputFile = fs.openSync(githubOutput, 'a');
+          fs.writeSync(outputFile, `repositories=${JSON.stringify(repositories)}`);
+          fs.closeSync(outputFile);
```

```suggestion
          fs.appendFileSync(githubOutput, `repositories=${JSON.stringify(repositories)}`);
```
Could this been combined into one line by using [`appendFileSync`](https://nodejs.org/api/fs.html#fsappendfilesyncpath-data-options) or that not equivalent?

---

Sorry my review was outdated. You can ignore it in favour of @koesie10's review.